tests/test_main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from unittest import mock
+from app.main import cryptocurrency_action
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_prediction_is_bullish(mocked_prediction_rate):
+    mocked_prediction_rate.return_value = 105.01
+    assert cryptocurrency_action(100) == "Buy more cryptocurrency"
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_prediction_is_bearish(mocked_prediction_rate):
+    mocked_prediction_rate.return_value = 94.99
+    assert cryptocurrency_action(100) == "Sell all your cryptocurrency"
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_prediction_is_in_range_more_or_less_5_percent(mocked_prediction_rate):
+    mocked_prediction_rate.return_value = 10
```

Comment:
Add boundary conditions, you don't check when `get_exchange_rate_prediction` return value equals exactly 1.05 or 0.95 * value passed in `cryptocurrency_action`.